setup.py

```diff
@@ -18,6 +20,34 @@
 from yamllint import linter
 
 
+def find_dirs(base_dir, exclude_dirs, include_dirs, dir_name):
```

Comment:
What's the use case of such complex `find_dirs` function?
Looking through the existing code it seems we default to empty excludes and include the `roles` subdir, which doesn't seem to justify this much code.